packages/kit/src/core/sync/write_tsconfig.js

```diff
@@ -123,7 +130,10 @@ export function write_tsconfig(kit, cwd = process.cwd()) {
 					lib: ['esnext', 'DOM', 'DOM.Iterable'],
 					moduleResolution: 'node',
 					module: 'esnext',
-					target: 'esnext'
+					target: 'esnext',
+
+					// TODO(v2): use the new flag verbatimModuleSyntax instead (requires support by Vite/Esbuild)
+					ignoreDeprecations: Number(ts.version.split('.')[0]) >= 5 ? '5.0' : undefined
```

Comment:
could we ever end up here if typescript isn't installed (in which case `ts.version` would blow up)?

Reply:
good point, AFAIK tsconfig is always written, even without TS installed, so we should check that